lib/vdsm/network/netinfo/bonding.py

```diff
@@ -18,35 +18,22 @@
 # Refer to the README and COPYING files for full details of the license
 from __future__ import absolute_import
 from __future__ import division
-from functools import partial
 import os
 
 import six
 
-from vdsm.network.ipwrapper import Link
-
-from .misc import visible_devs
-
 from vdsm.network.link.bond import Bond
 
 # In order to limit the scope of change, this module is now acting as a proxy
 # to the link.bond.sysfs_options module.
 from vdsm.network.link.bond import sysfs_options
-from vdsm.network.link.bond.sysfs_options import getDefaultBondingOptions
-from vdsm.network.link.bond.sysfs_options import getAllDefaultBondingOptions
-from vdsm.network.link.setup import parse_bond_options
 
-getDefaultBondingOptions
-getAllDefaultBondingOptions
-parse_bond_options
 
```

Comment:
It's also needed to remove usages of these indirections.

Reply:
done